assets/scss/base/_buttons.scss

```diff
@@ -99,7 +99,10 @@ button, .button {
 
   @include media(mobile) {
     margin-top: 10px;
+    text-align: center;
+    box-sizing: border-box;
```

Comment:
Our `.button.button--link` elements are 100% width in mobile. On all inputs, `box-sizing` is set to `border-box`. This means buttons sit nicely within the content area and don't spill outside. The link, however, spills out to the full viewport width and is therefore not aligned with the button. See below:

![screen shot 2016-03-01 at 4 31 43 pm](https://cloud.githubusercontent.com/assets/1764083/13433731/1dfa2b16-dfcb-11e5-937e-b1635359a334.png)

Settings box sizing on the link gives us what we need:

![screen shot 2016-03-01 at 4 33 10 pm](https://cloud.githubusercontent.com/assets/1764083/13433783/485418ea-dfcb-11e5-83bd-492b046be076.png)